src/main/java/com/google/step/youtube/Channel.java

```diff
@@ -0,0 +1,60 @@
+package com.google.step.youtube;
+
+import java.util.Date;
+
+/**
+ * Object for a YouTube channel and information on its videos. 
```

Comment:
nit: Instead of saying "Object" say "Value type for a YouTube ..."

Reply:
Changed.

Reply:
not a big deal, but did you actually change the javadoc?

Reply:
yep it's changed in pull #27